.gitignore

```diff
@@ -1,4 +1,6 @@
 /gitlab-cli
 /gitlab-cli_v*
```

Comment:
Those should then be removed, shouldn't they?

Reply:
Yep, I removed them.